ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/resources/streaming/TopicStream.java

```diff
@@ -32,46 +31,52 @@
 import java.io.StringWriter;
 import java.text.DateFormat;
 import java.text.SimpleDateFormat;
+import java.util.Arrays;
 import java.util.Date;
 import java.util.List;
 import java.util.Locale;
 import java.util.Objects;
+import java.util.Optional;
 import java.util.stream.Collectors;
 import java.util.stream.StreamSupport;
 
 import io.confluent.kafka.schemaregistry.client.SchemaRegistryClient;
 import io.confluent.kafka.serializers.KafkaAvroDeserializer;
+import io.confluent.ksql.rest.util.JsonMapper;
 import io.confluent.ksql.util.SchemaUtil;
 
-import static io.confluent.ksql.rest.server.resources.streaming.TopicStream.Format.getFormatter;
+public final class TopicStream {
 
-public class TopicStream {
+  private TopicStream() {
+  }
 
   public static class RecordFormatter {
 
     private static final Logger log = LoggerFactory.getLogger(RecordFormatter.class);
-    private final SchemaRegistryClient schemaRegistryClient;
+
+    private final KafkaAvroDeserializer avroDeserializer;
     private final String topicName;
+    private final DateFormat dateFormat =
+        SimpleDateFormat.getDateTimeInstance(3, 1, Locale.getDefault());
 
-    private Format format = Format.UNDEFINED;
+    private Formatter formatter = null;
```

Comment:
nit: null is not required. 4 characters takes up too much room on the screen ;-)

Reply:
Where you just trying to find something to complain about? ;)